sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeCustomFormsOperation.cs

```diff
@@ -51,11 +52,13 @@ public class RecognizeCustomFormsOperation : Operation<IReadOnlyList<RecognizedF
         /// <summary>
         /// </summary>
         /// <param name="operations"></param>
+        /// <param name="diagnostics"></param>
         /// <param name="modelId"></param>
         /// <param name="operationLocation"></param>
-        internal RecognizeCustomFormsOperation(ServiceClient operations, string modelId, string operationLocation)
+        internal RecognizeCustomFormsOperation(ServiceClient operations, ClientDiagnostics diagnostics, string modelId, string operationLocation)
```

Comment:
Would it make sense to pass client here too to be consistent across constructors?  Since this one is internal, it's not a big deal.  But wondering if it would make sense to keep the client as the private member and then call client.ServiceClient, client.Diagnostics on it instead of keeping both.  Small suggestion though.

Reply:
Taking into account we'll remove the `modelId` parameter in a separate PR, the `internal` and `public` constructors would end up having the same signature: `(string, FormRecognizerClient)`. Other operations have the exact same issue.

We can keep the parameters inverted in one of them, but I think that's a bit error-prone.

Reply:
Yes, good point.  I had thought the `operationId` in the public constructor would differentiate them, but with the removal of `modelId`, this is no longer the case.